workloads/hyper-scale/common.sh

```diff
@@ -2,12 +2,13 @@
 set -x
 
 source env.sh
+TEMP_DIR=`mktemp -d`
```

Comment:
Let's remove the old export TEMP_DIR line (line #15) if we are moving it here

Reply:
Done, updated.